packages/react/src/generators/application/application.spec.ts

```diff
@@ -165,15 +155,6 @@ describe('app', () => {
       const tsConfig = readJson(appTree, 'apps/my-app/tsconfig.json');
       expect(tsConfig.extends).toEqual('../../tsconfig.base.json');
     });
-
-    it('should extend from root tsconfig.json when no tsconfig.base.json', async () => {
```

Comment:
It no longer works like that. It will just extract tsconfig.base when ti is missing. So I updated one of the tests.

Bur for some reason we retest everything twice (with nested dirs and without), even though most of what we test has nothing to do with nested dirs.